spec/forms/candidate_interface/other_qualification_details_form_spec.rb

```diff
@@ -123,6 +123,22 @@
 
           expect(invalid_award_year.errors.full_messages_for(:award_year)).not_to be_empty
         end
+
+        it 'is not valid if non integer is used' do
+          invalid_award_year = described_class.new(nil, nil, qualification_type: 'Other', award_year: '2022.9')
+
+          invalid_award_year.valid?(:details)
+
+          expect(invalid_award_year.errors.full_messages_for(:award_year)).to eq(['Award year Year qualification awarded must be a real year'])
+        end
+
+        it 'is not valid if letters are used' do
+          invalid_award_year = described_class.new(nil, nil, qualification_type: 'Other', award_year: 'abcde')
+
+          invalid_award_year.valid?(:details)
+
+          expect(invalid_award_year.errors.full_messages_for(:award_year)).to eq(['Award year Year qualification awarded must be a real year', 'Award year Year qualification awarded must be this year, next year or a previous year'])
```

Comment:
Can we avoid both these error messages appearing at the same time? If the input isn't a "real year" (ie non-numeric) then the second error message should ideally not appear.

Reply:
I couldn't replicate both messages appearing on the frontend so thought it was fine. But I've added some code to avoid it appearing in the test, which is not ideal. thanks

Reply:
On the front end only the correct error message appeared? But not ideal for the test. Thanks